lib/rubocop/ast/node/mixin/method_dispatch_node.rb

```diff
@@ -153,14 +152,6 @@ def block_literal?
         parent&.block_type? && eql?(parent.send_node)
       end
 
-      # Checks whether this node is an arithmetic operation
-      #
-      # @return [Boolean] whether the dispatched method is an arithmetic
-      #                   operation
-      def arithmetic_operation?
```

Comment:
Same here. Perhaps that's used by some of RuboCop's extensions, plus it sounds like some useful method in general.

Reply:
checked in rubocop-rails and rubocop-performance and it is not being called, not sure about the other ones